manifests/kube-api.jsonnet

```diff
@@ -13,6 +13,12 @@ local kube = import "kube.libsonnet";
         resources: ["configmaps"],
         verbs: ["get", "list"],
       },
+      // Kubeapps creates Secrets with authorization token data for private chart repos
+      {
+        apiGroups: [""],
+        resources: ["secrets"],
+        verbs: ["get", "list", "create", "delete"],
```

Comment:
AFAIK we don't need delete because we use GC. Do we need get/list? Do we display the secret?

Reply:
we don't use the list and delete, but the Secret api has the methods implemented

Reply:
dropped all but `create` as thats the one we need

Reply:
> but the Secret api has the methods implemented
of course, but the point is to restrict any operations that aren't required. This way, Kubeapps doesn't have access to secrets it shouldn't have (technically Kubeapps already has cluster-wide access to read secrets, but this is not as important with the auth changes).